e2e/_screenshot.js

```diff
@@ -0,0 +1,81 @@
+"use strict";
```

Comment:
We decided to switch over to TS later rather than further delaying this feature.  Need to address the node typing issues with TS first.